src/utils/checkUserType.ts

```diff
@@ -0,0 +1,13 @@
+import { useLocalStorage } from 'usehooks-ts';
+
+const checkUserType = () => {
+  const [userType] = useLocalStorage('userType', '');
+
+  if (userType === 'company' || userType === 'customer') {
+    return true;
+  }
```

Comment:
p3: 이런식으로 많이 작성하기도 해요.

```suggestion
const VALID_USER_TYPE = ['company', 'customer'];

if (VALID_USER_TYPE.includes(userType)) {
  return true;
}
```

Reply:
@peakpay-leo 이 부분 어떻게 하면 차후 수정할 때 비용을 덜 들일 수 있을지 고민했었는데...! 좋은 지적 감사합니다 👏🏻